src/algebra/big_operators/basic.lean

```diff
@@ -865,15 +865,17 @@ lemma prod_dite_irrel (p : Prop) [decidable p] (s : finset α) (f : p → α →
   (∏ x in s, if h : p then f h x else g h x) = if h : p then ∏ x in s, f h x else ∏ x in s, g h x :=
 by { split_ifs with h; refl }
 
-@[simp] lemma sum_pi_single' {ι M : Type*} [decidable_eq ι] [add_comm_monoid M]
+@[simp, to_additive]
+lemma prod_pi_single' {ι M : Type*} [decidable_eq ι] [comm_monoid M]
```

Comment:
Can you remove the now `variables`-declared vairbles!

Reply:
Done